### PR TITLE
remove evals

### PR DESCRIPTION
**Pull request by Ferotiq**
### Description

Closes #

### CODE REVIEW

File: frontend/src/scripts/elements/commandline.ts [hunk 1]
```diff
@@ -668,17 +668,14 @@ $(document).on("click", "#keymap .r5 .key-space", () => {
...
+    $(event.currentTarget).attr("commands") as CommandlineLists.ListsObjectKeys
   );
-  const func = $(event.currentTarget).attr("function");
   if (commands !== undefined) {
     if ($(event.currentTarget).attr("commands") === "commandsTags") {
       CommandlineLists.updateTagCommands();
     }
     CommandlineLists.pushCurrent(commands);
     show();
-  } else if (func != undefined) {
```
Comment (Bruception) [inline frontend/src/scripts/elements/commandline.ts:680]: Is this logic being replaced?

Comment (Ferotiq) [reply frontend/src/scripts/elements/commandline.ts:680]: I don't think it's ever used right now. I did a find for a `function` attribute and didn't find it

Comment (Miodec) [reply frontend/src/scripts/elements/commandline.ts:680]: Blind mode used to use this i thiiink. Is it now just opening the commmands list?

Try enabling blind mode then click "blind mode" above the test words.

Comment (Miodec) [reply frontend/src/scripts/elements/commandline.ts:680]: Blind mode works fine

**Review by Miodec — APPROVED** [review]

**Pull request merged**